docs/Python-for-Formula-Authors.md

```diff
@@ -26,7 +26,9 @@ Applications should be installed into a Python [virtualenv](https://virtualenv.p
 
 All of the Python module dependencies of the application (and their dependencies, recursively) should be declared as `resource`s in the formula and installed into the virtualenv, as well. Each dependency should be explicitly specified; please do not rely on `setup.py` or `pip` to perform automatic dependency resolution, for the [reasons described here](Acceptable-Formulae.md#we-dont-like-install-scripts-that-download-unversioned-things).
 
-You can use [homebrew-pypi-poet](https://pypi.python.org/pypi/homebrew-pypi-poet) to help you write resource stanzas. To use it, set up a virtualenv and install your package and all its dependencies. Then, `pip install homebrew-pypi-poet` into the same virtualenv. Running `poet some_package` will generate the necessary resource stanzas. You can do this like:
+You can use `brew update-python-resources` to help you write resource stanzas. To use it, simply run `brew update-python-resources <formula>`. Sometimes, `brew update-python-resources` won't be able to automatically update the resources. If this happens, try running `brew update-python-resources --print-only <formula>` to print the resource stanzas instead of applying the changes directly to the file. You can then copy and paste resources as needed.
+
+If using `brew update-python-resources` doesn't work, you can use [homebrew-pypi-poet](https://pypi.python.org/pypi/homebrew-pypi-poet) to help you write resource stanzas. To use it, set up a virtualenv and install your package and all its dependencies. Then, `pip install homebrew-pypi-poet` into the same virtualenv. Running `poet some_package` will generate the necessary resource stanzas. You can do this like:
```

Comment:
Do we have any example formulae where we know `brew update-python-resources` does not work but `homebrew-pypi-poet` does not?

Reply:
Do you mean to ask for instances where `update-python-resources` fails but `poet` succeeds? If so, an example would be [`cloudformation-cli`](https://github.com/Homebrew/homebrew-core/blob/master/Formula/cloudformation-cli.rb). The formula specifies a specific `poet` command to use to generate the resources. The main feature difference that I can think of is that `poet` allows you to cherry-pick which packages to check.

If you mean examples where `update-python-resources` and `poet` both fail, `ansible` is a good example.

Reply:
Great, thanks!